### ISSUE TITLE
> Actually that's a good idea! It would help to be able to filter through the local pods timeline to find things that might be interesting to you when looking for new feeds to follo. What about just a temporary hide button?

### ISSUE THREAD

**Issue by prologic** [jointwt/twtxt#348]
> Actually that's a good idea! It would help to be able to filter through the local pods timeline to find things that might be interesting to you when looking for new feeds to follo. What about just a temporary hide button?￼

Temporary hide button is pretty cool thing! 
Also we can mute/hide twts _visually_ by dimming (css opacity: 15%) div-element of specified user twts!

__Originally posted by @ARyaskov in https://github.com/jointwt/twtxt/issues/344#issuecomment-764983828__

**Comment by sorenpeter** [comment]
could this be used for a way to mark twts as read? often i find myself opening up several tabs to the same conversation by looking thought the timetime of twtxt.net, because the new twts are not show in context. Not sure if this makes sense...

**Comment by der-lyse** [comment]
I don't think that this would work too well to simulate a read status, since it's only meant to be temporary. As far as I understood the original proposal, you would loose your "read status" when navigating away and back again. But this depends on the definition of "temporary".

I'd suggest implementing a real read status if you're after that. At least, that's one of the earliest features I've implemented in my own client, to keep track of conversations and hide already read twts/conversations.

**Comment by dbohdan** [comment]
You could hide a twt or multiple twts with a [`<details>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details) tag.

**Comment by prologic** [comment]
> You could hide a twt or multiple twts with a [`<details>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details) tag.

I _think_ given that the backend/frontend has a "pager" it would be _better_ to do temporary per-session filtering like what [Twet](https://github.com/quite/twet) can do. Although it just lets you select a feed source with `-s` -- Which we already have by just going to that source in the first place, be it an on-pod user or external feed.

**Comment by prologic** [comment]
> > You could hide a twt or multiple twts with a [`<details>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details) tag.
> 
> I _think_ given that the backend/frontend has a "pager" it would be _better_ to do temporary per-session filtering like what [Twet](https://github.com/quite/twet) can do. Although it just lets you select a feed source with `-s` -- Which we already have by just going to that source in the first place, be it an on-pod user or external feed.

Problem with this approach is the backend cache is not a database, it's just a cache. So you can't _really_ query or filter, but as it has. a limited size `N` the performance would be at worst `O(N)` to filter things out.

**Comment by prologic** [comment]
Just put up #399 which I _believe_ adds appropriate backend support for hiding Twts from your timeline based on Twter.URL. Basically you can hide twts from your timeline based on feed URIs you want to hide temporaily.

I just need someone better at UI/UX than me eo build a snazzy UI that makes sense here? This _is_ probably one of those times where a bit of JS would go nicely here to build up the `?hide=foo&hide=bar` list on the fly with little 'x''s to have a nice way to add to and remove the "hide" list.
